### DISCUSSION TITLE
Mundane warfare strange issues

### DISCUSSION THREAD

**Discussion by JohnRo79** [fedorovvl/tso_client#271] [Q&A] [answered]
here is where i am

on my island i just have my generals that i need for YWC
i'm creating my Mundane group and save the file
starting the adventure and then loading mundane file and sending gens
moving to adventure, i'm waiting for all my generals to land and then i do a F2 refresh
now i'm creating attack sequence (all hits from landing, no relocation)
saving my attack mundane file then loading it and hit Attack . all attacks go as scheduled

adventure ends, generals return, doing the same again : 
1.loading file, loading troops, sending to adventure
2.moving to adventure, waiting for all to land , hitting F2 refresh, loading file and attack
all goes well

but now the fun part starts

if i do the 3rd adventure, file works ok on my island , to load and send. no issues there
but on adventure, the generals land in different locations and file won't work

if i'll wait a day or so and try again, file works again , with generals landing same place
and then after my 2nd or 3rd attempt, generals land in mixed placed and file won't work again

what am i doing wrong here?

**Comment by Catavasiya** [comment, marked as answer]
The arrival points of generals on an adventure depend on many factors. Do not rely on their stability when repeating the adventure. For the attack template from the arriving zone first remove all arrived generals to the star menu, then place them on the available landing points  and from these points you can record the template.

**Comment by JohnRo79** [comment]
so, after all are landed, move them all in star
then land them all 
make my attack sequence 
and then save then load/use to attack ?

**Reply by Catavasiya** [reply]
yes. going AT with recorded template: you must wait for the arrival of all the generals, send them to the star menu and load the template: 1) to place generals, 2) to load army and 3) to attack.
U can create shortcuts like this:
![image](https://github.com/fedorovvl/tso_client/assets/128522616/5adb1cc6-92a6-4446-98db-2cb2555e07eb)

**Comment by JohnRo79** [comment]
i haven't done any template to place generals yet
is there a mass command? will they always land same place?

**Reply by Catavasiya** [reply]
"Return all generals to star menu" in my screenshot - mass command.
for landing all generals in stable places you have to record template. mundane template contains  1) points from which generals attack 2) army of generals 3) targets for attacks and timings between generals.

here is discord server link for discussions/help etc https://discord.gg/the-settlers-fans-662700970857332786

**Comment by JohnRo79** [comment]
all sorted now
sooooo easy, tyvm

now all i have to do is learn how to share my guide on discord
